### ISSUE TITLE
teleport with location name

### ISSUE THREAD

**Issue by sys0pt** [orangeadam3/terra121#22]
Please implement teleportation with city names additional to long/lat teleportation

Maybe: /tpgeo *what you would put in search bar*

**Comment by orangeadam3** [comment]
Yeah i think terrarium does this. I could look into it further down the line but probably not for a while.

**Comment by mreel** [comment]
I think this would be great, that being said, I also think it should be low priority. It's easy enough to look up the coordinates of a city, on top of that, a lot of city names exist multiple times, so dealing with that probably isn't the easiest thing.
I believe making the terrain better/more accurate and performance/stability will make a much bigger difference in overall quality of the mod.
